cli/crates/server/src/servers.rs

```diff
@@ -105,6 +105,8 @@ pub async fn start(
 ) -> Result<(), ServerError> {
     let project = Project::get();
 
+    // Exporting Pathfinder, TS parser & miniflare for resolvers.
+    export_embedded_files()?;
```

Comment:
@Finistere IIRC we said we'd check if TS config is in use first no?

Reply:
we need Pathfinder in all cases